bleep-core/src/scala/bleep/internal/ScriptShelloutCommand.scala

```diff
@@ -27,7 +27,7 @@ object ScriptShelloutCommand {
   def getForScript(scriptDef: ScriptDef, bleepExecutable: BleepExecutable): List[String] = scriptDef match {
     case Main(project, main, _) =>
       val bleepPath = bleepExecutable.getPath().toString
-      List(bleepPath, "run", "--main", main, project.name.value)
+      List(bleepPath, "run", "--class", main, project.name.value)
```

Comment:
it'd be great if this was aligned with `scala-cli`'s   `-M, --main-class` 

Reply:
Agreed! Feel free to change it everywhere, now or in a follow-up